xskillscore/core/comparative.py

```diff
@@ -152,16 +152,19 @@ def _categorical_metric(observations, forecasts, dim):
             metric_f1o = -metric_f1o
             metric_f2o = -metric_f2o
 
-    sign_test = (1 * (metric_f1o < metric_f2o) - 1 * (metric_f2o < metric_f1o)).cumsum(
+    walk = (1 * (metric_f1o < metric_f2o) - 1 * (metric_f2o < metric_f1o)).cumsum(
         time_dim
     )
 
-    # Estimate 95% confidence interval -----
+    # Estimate 1 - alpha confidence interval -----
     notnan = 1 * (metric_f1o.notnull() & metric_f2o.notnull())
     N = notnan.cumsum(time_dim)
+    # convert N to DataArray to use as coordinate
+    if isinstance(N, xr.Dataset):
```

Comment:
What about if there are multiple variables in the input Dataset? E.g., if I wanted to perform the sign test for variable `a` and `b` simultaneously. Would `sign_test` work in this case? I _think_ everything would work here except for adding confidence as a coordinate.

Perhaps it is easiest just to have `walk` and `confidence` returned as separate variables after all?... Or, at the very least, we should catch instances where multi-variable Datasets are provided and throw an error?

Reply:
UPDATE: Or, even easier, why not use the `_add_as_coord()` function in `utils`, which should deal with multi-variable Datasets appropriately.

Reply:
I dont want to have the `coordinate_suffix`